src/jaxsim/mujoco/loaders.py

```diff
@@ -567,4 +585,26 @@ def convert(
             considered_joints=considered_joints,
             plane_normal=plane_normal,
             heightmap=heightmap,
+            cameras=cameras,
         )
+
+
+class _Camera:
+    def __init__(self, name: str, mode: str, pos: str, xyaxes: str, fovy: str):
+        self.name = name
+        self.mode = mode
+        self.pos = pos
+        self.xyaxes = xyaxes
+        self.fovy = fovy
```

Comment:
At this point, why not the following?

```suggestion
@dataclasses.dataclass
class MujocoCamera:
    name: str
    mode: str
    pos: str
    xyaxes: str
    fovy: str
```

Reply:
You can then use `dataclasses.asdict(dc)`.

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/133/commits/702af250ca674950bb0c54df888cfa1531a461e1